rm-datasource/src/main/java/io/seata/rm/datasource/StatementProxy.java

```diff
@@ -113,6 +114,16 @@ public boolean execute(String sql, String[] columnNames) throws SQLException {
         return ExecuteTemplate.execute(this, (statement, args) -> statement.execute((String) args[0],(String[])args[1]), sql,columnNames);
     }
 
+    @Override
+    public void addBatch(String sql) throws SQLException {
+        if (StringUtils.isNotBlank(targetSQL)) {
+            targetSQL += "; " + sql;
```

Comment:
If the business sql has ";", the spliced sql will be wrong.
But the business side can handle it, and it can be changed here or not.

Reply:
There may be problems using split characters here. The best strategy is to store sql in List<String>

Reply:
> There may be problems using split characters here. The best strategy is to store sql in List

Released soon, no major changes.